style.css

```diff
@@ -331,3 +345,96 @@
   height: 80px;
   border-radius: 50px;
 }
+
+.contact-form {
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  margin: 10px 0;
+}
+
+.contact-form .contact-header {
+  width: 80%;
+  margin: auto;
+  text-align: center;
+}
+
+.contact-form .contact-header .contact-info {
+  color: #172b4d;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 57px;
+}
+
+.contact-form .form-fields {
+  width: 90%;
+}
+
+.contact-form .form-fields .list {
+  list-style: none;
+}
+
+.contact-form .form-fields .list input {
+  width: 100%;
+  height: 48px;
+  border: 1px solid grey;
+  margin: 10px 0;
+  border-radius: 5px;
+  padding: 10px;
+}
+
+.contact-form .form-fields .list textarea {
+  width: 100%;
+  border: 1px solid grey;
+  height: 114px;
+  margin: 10px 0;
+  border-radius: 5px;
+  padding: 10px;
+}
```

Comment:
- [ ] Kindly add green color to the border of the message area as shown in the figma template. That would increase user experience.